test_scripts/AppServices/GetAppServiceRecords/001_GetAppServiceRecords_MOBILE.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) app1 is Registered
+--  2) AppServiceProvider permissions are assigned for <app1ID>
+--
+--  Steps:
+--  1) app1 sends a PublishAppService (with {serviceType="MEDIA", handledRPC=ButtonPress})
+--  2) HMI sends a GetAppServiceRecords to Core
+--
+--  Expected:
+--  1) Core returns the service record of the MEDIA service published by app1 ({servicePublished=true, serviceActive=true}) 
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/AppServices/commonAppServices')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local functions ]]
+local function PTUfunc(tbl)
+    local pt_entry = common.getAppServiceProducerConfig(1)
+    pt_entry.app_services.MEDIA = { handled_rpcs = {{function_id = 41}} }
```

Comment:
Did you mean to overwrite the service name here?

Reply:
pt_entry.app_services.MEDIA.handled_rpcs = {{function_id = 41}} ?

Reply:
whoops... not intentional. Carried over my changes from RPC passing where I had to define a new NAVIGATION section. I'll commit this fix